### PR TITLE
Add a bunch of additional shorteners

### PR DESCRIPTION
**Pull request by rroller**
Add a bunch of additional shorteners

Many of these are for pure abuse (not all, but some).

I left out `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.com` because it's the inverse. But let me know if you want it.

### CODE REVIEW

**Review by PeterDaveHello — CHANGES_REQUESTED** [review]
Looks like there are some not resolving domains, like:

- 1wb2.net
- 2ad.in
- 4view.me
- 5em.cz
- 6fr.ru
- 76.gd
- 77.ai

(Only active url-shorteners will be listed here.)

**Review by PeterDaveHello — CHANGES_REQUESTED** [review]
Looks like some of them are parked domain, should be removed, as we only list active url shorteners for the best accuracy.

Some example of parked ones:

- 1link.in
- 1tk.us
- 1url.com
- 2big.at
- 2ty.in
- 3ra.be
- 4i.ae
- 4url.cc
- 5url.net
- 5z8.info
- 7fth.cc
- 944.la
- 98.to
- abbrr.com
- adfa.st
- adfly.fr
- adv.li
- ajn.me
- a.nf
- asso.in
- baid.us
- bee4.biz
- bloat.me
- bote.me
- brk.to
- brzu.net
- burnurl.com
- captur.in
- catchylink.com
- cf6.co
- clikk.in

**Comment by PeterDaveHello** [comment]
Just curious, so I did some additional check, more than one hundred parking domain in it 😅 , just FYI.

- cutt.eu
- cuturl.com
- d75.eu
- duff-links.com
- dwarfurl.com
- esyurl.com
- ex9.co
- ezurl.cc
- foe.hn
- fur.ly
- hadej.co
- hec.su
- hex.io
- hide.my
- hops.me
- href.in
- hurl.me
- iiiii.in
- iky.fr
- ilix.in
- isra.li
- iterasi.net
- itm.im
- its.gd
- jijr.com
- jp22.net
- just.as
- kl.am
- kxb.me
- lcut.in
- l.gg
- linkn.co
- link.sv
- llu.ch
- lnk.in
- lt.tl
- mcaf.ee
- minilien.com
- miniurl.com
- ne1.net
- nig.gr
- njx.me
- nov.io
- nsfw.in
- past.is
- pdh.co
- ph.dog
- pich.in
- pin.st
- piurl.com
- po.st
- posted.at
- post.ly
- ptab.it
- qbn.ru
- qxp.sk
- rb6.me
- rdz.me
- redir.fr
- redirx.com
- redu.it
- ref.so
- ri.ms
- s7y.us
- safelink.live
- safe.mn
- sharetabs.com
- short.pk
- short.to
- shoturl.us
- shrinkster.com
- shrten.com
- shrt.in
- shw.me
- simurl.com
- sk.gy
- smll.co
- snsw.us
- sn.vc
- spn.sr
- sqrl.it
- ssl.gs
- sturly.com
- s-url.fr
- sy.pe
- ta.gd
- tb.cn
- tighturl.com
- tini.cc
- tiny123.com
- tiny.ms
- tldr.sk
- tnij.org
- tr5.in
- traceurl.com
- trick.ly
- tr.im
- twitterpan.com
- twurl.cc
- ucam.me
- u.mavrev.com
- unlc.us
- updating.me
- upzat.com
- ur1.ca
- url5.org
- url.co.uk
- urlcover.com
- urlenco.de
- urlhawk.com
- urlot.com
- urub.us
- vaza.me
- vgn.me
- watchme.ml
- wed.li
- wwy.me
- xib.me
- xrl.in
- xua.me
- xub.me
- yagoa.fr
- yect.com
- yep.it
- yzh.li
- za.uy
- zi.pe
- zipmyurl.com
- zoodl.com
- zpag.es

**Comment by PeterDaveHello** [comment]
@rroller with all due respect, not sure if you have any feedbacks? 😅

**Comment by PeterDaveHello** [comment]
BTW I just added a new list called `inactive` for those domains currently used as url shortener. If there's no further updates on this issue, will close it later. Thanks anyway!

**Comment by rroller** [comment]
I collected these, I have insights to gather them, but wont' mention how, but it's my data. Feel free to take them or leave them. Many of them are short lived, but not all.
Feel free to go through and pick the ones that you want.

**Comment by PeterDaveHello** [comment]
@rroller Okay, thanks!

File: list [hunk 1]
```diff
@@ -335,294 +686,801 @@ lihi2.me
...
+linkbun.ch
 linkd.pl
+linkjust.com
+linkn.co
+linkr.bio
 linkr.in
+linkr.it
 links2.me
 linkshare.pro
+linksly.co
```
Comment (ignoramous) [inline list:711]: `linklyhq.com` as well?

**Comment by samuveth** [comment]
We are missing clc.to
